examples/custom-esbuild/sanity-esbuild-app/angular.json

```diff
@@ -0,0 +1,126 @@
+{
+  "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
```

Comment:
Looks good, do you think we need to add another sanity app which is ESM by default (like [here](https://github.com/just-jeb/angular-builders/blob/master/examples/custom-webpack/sanity-app-esm/angular.json))?